meeshkan/serve/mock/specs.py

```diff
@@ -0,0 +1,43 @@
+import os
+import logging
+from openapi_typed_2 import (
+    convert_to_openapi,
+    OpenAPIObject,
+)
+from typing import List
+import json
+import yaml
+from dataclasses import dataclass
+
+
+@dataclass
+class OpenAPISpecification:
```

Comment:
Generic nitpicking: I think OpenAPI specification is the specification that all OpenAPI documents should conform to. Like "a document must have key 'openapi'". Not related to this PR of course, just something I've been thinking about :)

Reply:
Yea, naming is hard! Do you have a suggestion for what to name this construct? I had `OpenAPIWithSource` earlier.

Reply:
No I don't :) `OpenAPIWithSource` works well, at least until we need to add another field. Maybe `OpenAPIWrapper`, `APIInfo`? Maybe something that somehow reflects the fact the document has a source, like it's some kind of result of load... I don't know, OpenAPISpecification works fine!